core/src/main/scala/kafka/network/RequestChannel.scala

```diff
@@ -432,43 +432,6 @@ class RequestChannel(val queueSize: Int,
     }
   }
 
-  def sendResponse(request: RequestChannel.Request,
```

Comment:
In the trunk patch, we ended up moving these two methods out of RequestChannel in order to keep the test mocks happy